addons/mail/static/src/messaging/entity/entity_core.js

```diff
@@ -11,94 +11,58 @@ const registry = {};
 
 /**
  * @private
- * @param {string} entityName
- * @returns {Object}
- */
-function _getEntryFromEntityName(entityName) {
-    if (!registry[entityName]) {
-        registry[entityName] = {
-            dependencies: [],
-            factory: undefined,
-            name: entityName,
-            patches: [],
-        };
-    }
-    return registry[entityName];
-}
-
-/**
- * @private
- * @param {string} entityName
- * @param {string} patchName
- * @param {Object} patch
- * @param {Object} [param3={}]
- * @param {string} [param3.type='instance'] 'instance', 'class' or 'relation'
- */
-function _registerPatchEntity(entityName, patchName, patch, { type = 'instance' } = {}) {
-    const entry = _getEntryFromEntityName(entityName);
-    Object.assign(entry, {
-        patches: (entry.patches || []).concat([{
-            name: patchName,
-            patch,
-            type,
-        }]),
-    });
-}
```

Comment:
I hate seeing this whole diff just because of a rename, this is not reviewable... I think we should do a prep move commit when that happens (or a post move commit just for alphabetical order).